### DISCUSSION TITLE
Question: Grabbing Max Coordinates

### DISCUSSION THREAD

**Discussion by metacode64** [NicoHood/HID#280] [Q&A]
Hello,

I just stumbled across this library and its great.I found the absolute mouse example and it looks good. It says the absolute coordinates are for the desktop, I am wondering is there a command to find the max absolute x and y coordinates of the desktop so that the mouse can be properly aligned, or does Windows automatically scale the resolution of the absolute mouse?

**Comment by NicoHood** [comment]
The min/max is stated here:
https://github.com/NicoHood/HID/blob/master/examples/Mouse/AbsoluteMouse/AbsoluteMouse.ino#L49

Windows will map that to your monitor.
